### PR TITLE
Changed:

### PR DESCRIPTION
**Pull request by maximVorochaev**
- security DB scheme

### CODE REVIEW

File: filethread-db/src/main/resources/db/changelog/db.changelog-1.0.xml [hunk 1]
```diff
@@ -6,119 +6,17 @@
...
-                <column name="updated" type="TIMESTAMP" defaultValue="now()">
-                    <constraints nullable="false"/>
-                </column>
-            </createTable>
-        </changeSet>
-    <changeSet id="1.0.7" author="mvorochaev">
-        <createTable tableName="chat_room">
-            <column name="id" type="SERIAL" autoIncrement="true">
+    <changeSet id="1.0.0" author="mvorochaev">
+        <createTable tableName="person">
```
Comment (IharYudziankou) [inline filethread-db/src/main/resources/db/changelog/db.changelog-1.0.xml:10]: Is **user** better name ?

File: filethread-db/src/main/resources/db/changelog/db.changelog-1.0.xml [hunk 2]
```diff
@@ -6,119 +6,17 @@
...
+            <column name="id" type="BIGSERIAL" autoIncrement="true">
                 <constraints primaryKey="true" nullable="false"/>
             </column>
-            <column name="status" type="VARCHAR(30)" defaultValue="ACTIVE">
+            <column name="first_name" type="VARCHAR(100)"/>
+            <column name="last_name" type="VARCHAR(100)"/>
+            <column name="email" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
```
Comment (IharYudziankou) [inline filethread-db/src/main/resources/db/changelog/db.changelog-1.0.xml:18]: unique constraint should be added

**Review by IharYudziankou — CHANGES_REQUESTED** [review]

**Comment by IharYudziankou** [comment]
Let's add rollback scripts also.

File: filethread-db/src/main/resources/db/changelog/db.changelog-1.0.xml (hunk 1 above)
Comment (maximVorochaev) [reply filethread-db/src/main/resources/db/changelog/db.changelog-1.0.xml:10]: no

File: filethread-db/src/main/resources/db/changelog/db.changelog-1.0.xml (hunk 2 above)
Comment (maximVorochaev) [reply filethread-db/src/main/resources/db/changelog/db.changelog-1.0.xml:18]: ready

**Review by maximVorochaev — COMMENTED** [review]
ready

**Review by PavelEvdokimovich — APPROVED** [review]

**Review by IharYudziankou — APPROVED** [review]

**Pull request merged**